### PR TITLE
feat(SidebarE): Add design documentation

### PR DESCRIPTION
**Pull request by hxltrhuxze**
Adds sidebarE design documentation

### CODE REVIEW

**Review by m7kvqbe1 — APPROVED** [review]

File: packages/docs-site/src/library/pages/components/sidebar_experimental.md [hunk 1]
```diff
@@ -2,44 +2,51 @@
...
 <TabSet>
 <Tab title="Design">
 
 ### Anatomy
 
-TBC
+<SidebarEAnatomy />
 
-### Sizing & Spacing
+The sidebar component has two main sections - the main navigational area at the top of the bar, and the fixed area at the bottom of the bar.
```
Comment (thyhjwb6) [inline packages/docs-site/src/library/pages/components/sidebar_experimental.md:33]: Should this be `Sidebar`?

File: packages/docs-site/src/library/pages/components/sidebar_experimental.md [hunk 2]
```diff
@@ -2,44 +2,51 @@
...
 # Overview
 
 The Sidebar is a navigational component. It is fixed to the left-hand of the screen and extends the full height of the browser. This component stays in place whilst the application scrolls, ensuring top navigational items are always within the user's reach.
 
+<SidebarEComponent />
+
 ## Usage
 
-TBC
+Only one Sidebar component should be used per page. It should contain your top level navigational items, not sub-navigational items/actions for a single page.
```
Comment (thyhjwb6) [inline packages/docs-site/src/library/pages/components/sidebar_experimental.md:24]: Should this be `top-level`?

File: packages/docs-site/src/library/pages/components/sidebar_experimental.md [hunk 3]
```diff
@@ -2,44 +2,51 @@
...
 ### Anatomy
 
-TBC
+<SidebarEAnatomy />
 
-### Sizing & Spacing
+The sidebar component has two main sections - the main navigational area at the top of the bar, and the fixed area at the bottom of the bar.
 
-TBC
+  2. **Collapse Toggle**. Toggles the sidebar between the collapsed and expanded views.
```
Comment (thyhjwb6) [inline packages/docs-site/src/library/pages/components/sidebar_experimental.md:35]: Again `S`

**Review by thyhjwb6 — CHANGES_REQUESTED** [review]
Looks good 👍 

There seems to be a mix of `top level` vs `top-level` and `the sidebar` vs `the Sidebar`. I'd also consider treating the other component names as nouns and therefore capitalising.

**Pull request merged**